### PR TITLE
Address my todos

### PR DESCRIPTION
**Pull request by cgreene**
I took a pass through the document in an attempt to address the todos that I was most well suited to address. Primarily this was a focus on the contributor expectations discussion, but I also touched on some of the others that needed independent confirmation.

### CODE REVIEW

File: content/02.main-text.md [hunk 1]
```diff
@@ -86,7 +83,14 @@ In addition, git carefully tracks all revisions to the manuscript text but not t
 These discussions must be archived to ensure that important decisions about the manuscript are preserved and authors receive credit for intellectual contributions that are not directly reflected in the manuscript's text.
 GitHub supports programmatic access to issues, pull requests, and reviews so tracking these conversations is feasible in the future.
 
-`TODO: cgreene paragraph on diversity, open science culture, possibly author disagreements and conduct`
+It is important to consider how the move to an open contribution model affects under-represented minority members of the scientific community [@url:https://lgatto.github.io/open-and-open/].
+Recent work has identified clear social norms and processes as helpful to maintaining a collaborative culture [@doi:10.1080/08874417.2009.11645338].
```
Comment (dhimmel) [inline content/02.main-text.md]: `10.1080/08874417.2009.11645338` is not a registered DOI

File: content/02.main-text.md [hunk 2]
```diff
@@ -14,10 +14,8 @@ However, inviting wide authorship brings many technical and social challenges su
 
 We present solutions to these challenges based on our recent experience leading a collaborative review "Opportunities And Obstacles For Deep Learning In Biology And Medicine" [@doi:10.1101/142760].
 Our review attracted 27 authors from 20 different institutions who were not determined in advance.
-`TODO: confirm institution count`
 We wrote entirely in the open without restrictions on who was welcome to contribute.
-Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media and independently decided to contribute.
-`TODO: confirm "most"`
+Although we requested that some authors participate for their specific expertise, most seem to have discovered the manuscript organically through conferences or social media and independently decided to contribute.
```
Comment (dhimmel) [inline content/02.main-text.md]: In the name of persuasive writing, I think we can strip "seem to have"

File: content/02.main-text.md [hunk 3]
```diff
@@ -86,7 +83,14 @@ In addition, git carefully tracks all revisions to the manuscript text but not t
...
 GitHub supports programmatic access to issues, pull requests, and reviews so tracking these conversations is feasible in the future.
 
-`TODO: cgreene paragraph on diversity, open science culture, possibly author disagreements and conduct`
+It is important to consider how the move to an open contribution model affects under-represented minority members of the scientific community [@url:https://lgatto.github.io/open-and-open/].
+Recent work has identified clear social norms and processes as helpful to maintaining a collaborative culture [@doi:10.1080/08874417.2009.11645338].
+Conferences and open source projects have used codes of conduct to establish these norms [@url:http://geekfeminism.wikia.com/wiki/Code_of_conduct_evaluations?oldid=35801; @url:https://www.contributor-covenant.org/].
+In our open review paper, we established contributor guidelines that discussed norms in the areas of of text contribution, peer review, and authorship [@url:https://github.com/greenelab/deep-review/blob/2bb8c3960bc4616f32eaceb3fe06002c51fb2784/CONTRIBUTING.md], which we identified in advance as potential areas of disagreement.
```
Comment (dhimmel) [inline content/02.main-text.md]: I would link contributor guidelines rather than do a URL citaiton.

File: content/02.main-text.md [hunk 4]
```diff
@@ -86,7 +83,14 @@ In addition, git carefully tracks all revisions to the manuscript text but not t
 These discussions must be archived to ensure that important decisions about the manuscript are preserved and authors receive credit for intellectual contributions that are not directly reflected in the manuscript's text.
 GitHub supports programmatic access to issues, pull requests, and reviews so tracking these conversations is feasible in the future.
 
-`TODO: cgreene paragraph on diversity, open science culture, possibly author disagreements and conduct`
+It is important to consider how the move to an open contribution model affects under-represented minority members of the scientific community [@url:https://lgatto.github.io/open-and-open/].
```
Comment (dhimmel) [inline content/02.main-text.md]: I'm not sure how the contributor guidelines discussed below relate to making the open contribution model equitable for URMs. In what way did the deep review consider this?

**Update**, I hadn't finished reading this paragraph:

> Our contributor guidelines did not discuss broader community norms that may have improved inclusiveness

Would it make sense to put all this content together? For example, start with what our contributing guidelines did address. Then say what we recommend future guidelines also address.

**Review by dhimmel — COMMENTED** [review]
Just some minor comments.

File: content/02.main-text.md (hunk 1 above)
Comment (dhimmel) [reply content/02.main-text.md]: Interesting, this article is titled "The Role of Wiki Technology and Altruism in Collaborative Knowledge Creation" and was published in 2009 in the _Journal of Computer Information Systems_. The publisher's site is http://www.tandfonline.com/doi/abs/10.1080/08874417.2009.11645338, which states that the DOI is `10.1080/08874417.2009.11645338`, however this is not a registered DOI. In addition, the title is not registered with Crossref, although a precursor of the article does have a valid DOI: https://doi.org/10.1109/hicss.2007.277. 

I [tweeted](https://twitter.com/dhimmel/status/916024670129762304) Taylor & Francis to fix this.

Comment (dhimmel) [reply content/02.main-text.md]: Options are:

+ do not cite
+ cite with DOI, but manually override JSON CSL. Will result in broken DOI link that may resolve in the future
+ cite with URL
+ cite `@doi:10.1109/hicss.2007.277` instead

File: content/02.main-text.md (hunk 2 above)
Comment (agitter) [reply content/02.main-text.md]: Agree we can remove "seem to have".  @cgreene and I discussed this after his commit.  Our author counts are fuzzy but do support "most".

File: content/02.main-text.md [hunk 5]
```diff
@@ -86,7 +83,14 @@ In addition, git carefully tracks all revisions to the manuscript text but not t
...
+Recent work has identified clear social norms and processes as helpful to maintaining a collaborative culture [@doi:10.1080/08874417.2009.11645338].
+Conferences and open source projects have used codes of conduct to establish these norms [@url:http://geekfeminism.wikia.com/wiki/Code_of_conduct_evaluations?oldid=35801; @url:https://www.contributor-covenant.org/].
+In our open review paper, we established contributor guidelines that discussed norms in the areas of of text contribution, peer review, and authorship [@url:https://github.com/greenelab/deep-review/blob/2bb8c3960bc4616f32eaceb3fe06002c51fb2784/CONTRIBUTING.md], which we identified in advance as potential areas of disagreement.
+Our contributor guidelines required verifiable participation: either directly attributable changes to the text or participation in the discussion on GitHub.
+We maintained our guidelines, even when a case arose where two authors had written text together but only one had directly attributable changes and participation.
+Our contributor guidelines did not discuss broader community norms that may have improved inclusiveness.
```
Comment (agitter) [inline content/02.main-text.md]: "Our contributor guidelines" is repeated.  Maybe use "these guidelines" instead in this line or the previous instance?

File: content/02.main-text.md (hunk 3 above)
Comment (agitter) [inline content/02.main-text.md]: Was that particular commit significant?  We also have a tagged version https://github.com/greenelab/deep-review/blob/v0.9-preprint/CONTRIBUTING.md

**Review by agitter — CHANGES_REQUESTED** [review]
Looks good, only minor suggestions from me.

File: content/02.main-text.md (hunk 2 above)
Comment (cgreene) [reply content/02.main-text.md]: 👍

File: content/02.main-text.md (hunk 4 above)
Comment (cgreene) [reply content/02.main-text.md]: 👍 see if you like the revision

File: content/02.main-text.md (hunk 3 above)
Comment (cgreene) [reply content/02.main-text.md]: I went to the tagged one that @agitter suggested below. Let me know if you think something different would be better.

Comment (cgreene) [reply content/02.main-text.md]: 👍

Comment (cgreene) [reply content/02.main-text.md]: Also converted to link 👍

File: content/02.main-text.md (hunk 5 above)
Comment (cgreene) [reply content/02.main-text.md]: 👍

File: content/02.main-text.md (hunk 1 above)
Comment (cgreene) [reply content/02.main-text.md]: I did not address this one. @agitter : what is your preference here?

**Comment by cgreene** [comment]
Ok - I addressed everything except the DOI with issues. I'm not quite sure what to do there. I guess moving to the citation of the earlier paper seems like the cleanest way to go right now.

File: content/02.main-text.md [hunk 6]
```diff
@@ -86,7 +83,14 @@ In addition, git carefully tracks all revisions to the manuscript text but not t
...
 
-`TODO: cgreene paragraph on diversity, open science culture, possibly author disagreements and conduct`
+In our open review paper, we established [contributor guidelines](https://github.com/greenelab/deep-review/blob/v0.9-preprint/CONTRIBUTING.md) that discussed norms in the areas of of text contribution, peer review, and authorship, which we identified in advance as potential areas of disagreement.
+Our contributor guidelines required verifiable participation: either directly attributable changes to the text or participation in the discussion on GitHub.
+We maintained our guidelines, even when a case arose where two authors had written text together but only one had directly attributable changes and participation.
+These guidelines did not discuss broader community norms that may have improved inclusiveness.
+It is also important to consider how the move to an open contribution model affects under-represented minority members of the scientific community [@url:https://lgatto.github.io/open-and-open/].
+Recent work has identified clear social norms and processes as helpful to maintaining a collaborative culture [@doi:10.1080/08874417.2009.11645338].
```
Comment (dhimmel) [inline content/02.main-text.md]: > Ok - I addressed everything except the DOI with issues. I'm not quite sure what to do there. I guess moving to the citation of the earlier paper seems like the cleanest way to go right now.

What about switching to the previous version and commenting in the source:

```md
Recent work has identified clear social norms and processes as helpful to maintaining a collaborative culture [@doi:10.1109/hicss.2007.277].
<!-- cited a previous version of @doi:10.1080/08874417.2009.11645338 (see https://git.io/vdSSR) -->
```

**Review by dhimmel — APPROVED** [review]

File: content/02.main-text.md (hunk 6 above)
Comment (cgreene) [reply content/02.main-text.md]: I agree with this 👍  - done!

**Comment by cgreene** [comment]
I'm so confused. I just pushed changes but they're not showing up in the github interface...

File: content/02.main-text.md (hunk 6 above)
Comment (dhimmel) [reply content/02.main-text.md]: From @cgreene:

> manubot appears to try to build the @doi: etc even if it's in a comment
> https://travis-ci.org/greenelab/meta-review/builds/289122620

Yes I'll add HTML comments the the snippets that should be ignored in https://github.com/greenelab/manubot/issues/13.

Removing `@` or `@doi:` as was done in https://github.com/greenelab/meta-review/pull/12/commits/370e6d79be7cd387eac6c7be6fca8b94ecca4b04 is a good workaround.

File: content/02.main-text.md (hunk 1 above)
Comment (agitter) [reply content/02.main-text.md]: The solution proposed below looks good to me

**Review by agitter — APPROVED** [review]

**Comment by agitter** [comment]
Looks good to me.  I'll merge this and check the URL references to see if any need manual overrides.

**Pull request merged**